### ISSUE TITLE
Integrate single-provisioner consensus with Rusk service

### ISSUE THREAD

**Issue by goshawk-3** [dusk-network/dusk-blockchain#1215]
**Describe what you want implemented** 
Test-harness/Devnet should be capable of running single-provisioner consensus entirely driven by Rusk service.  The target branch of this task must be **testnet** branch.

In other words, on substituting Rusk mock with Rusk service and regenerating Genesis Block:

- any node in the network should be capable to call  **Accept/Finalize**.
- genesis provisioner should be capable to call both **ExecuteStateTransition** and **VerifyStateTransition**
- genesis provisioner should be capable of loading both Wallet and Consensus keys

Steps to follow:

- [x] Replace rusk mock with rusk service for test-harness (#1216)
- [x] Regenerate genesis block with one valid stake (Genesis Provisioner)
- [x] Regenerate genesis wallet
- [x] Refactor test-harness so that
    - it can load wallet and consensus keys. rusk.LoadWallet
    - run rusk service

**Describe "Why" this is needed**

Having **single-provisioner** consensus fully functioning in pair with latest Rusk service should serve as a backbone and prerequisite for completion of **multi-provisioners** consensus integration. This will allow us to maintain a running CI.  Any integration issue should be reported as a separate issue in any of dusk-blockchain, rusk or rusk-vm repositories . 

NB:  this task should go in parallel with Kadcast integration which is also prerequisite to multi-provisioners consensus.
